src/Searcher.h

```diff
@@ -145,6 +149,12 @@ class Searcher {
      *
      * \param[in] team the team that we belong to.
      */
+    struct ScoreSort {
```

Comment:
Are you using this/going to use this? Also, Node already has `operator>` and `operator<`.